src/java.base/share/classes/java/io/InputStreamReader.java

```diff
@@ -162,20 +162,10 @@ public int read() throws IOException {
     }
 
     /**
-     * Reads characters into a portion of an array.
-     *
-     * @param      cbuf     Destination buffer
-     * @param      offset   Offset at which to start storing characters
-     * @param      length   Maximum number of characters to read
-     *
-     * @return     The number of characters read, or -1 if the end of the
-     *             stream has been reached
-     *
-     * @throws     IOException  If an I/O error occurs
-     * @throws     IndexOutOfBoundsException {@inheritDoc}
+     * {@inheritDoc}
      */
-    public int read(char cbuf[], int offset, int length) throws IOException {
-        return sd.read(cbuf, offset, length);
+    public int read(char[] cbuf, int off, int len) throws IOException {
```

Comment:
IOOBE is unchecked, are you sure it gets inherited into the sub-class here?

Reply:
It does not due to https://bugs.openjdk.java.net/browse/JDK-8157677.

Reply:
The long standard behavior is that javadoc doesn't copy the  throws of unchecked exceptions, instead we've had to declare the throws and use inheritDoc. I assume you'll need to do that here.

Reply:
It does not and there is a javadoc issue already on file about this.